setup.py

```diff
@@ -16,11 +16,11 @@
 here = path.abspath(path.dirname(__file__))
 
 deps = [
-    "importlib-resources>=5.1,<5.2",
+    "importlib-resources>=5.4,<5.5",
     "numpy>=1.19,<1.20",
-    "pillow>=8.2,<8.3",
-    "tensorflow>=2.4,<2.5",
-    "tensorflowjs>=3.4,<3.5",
+    "pillow>=9.0,<9.1",
+    "tensorflow>=2.7,<2.8",
```

Comment:
Some mates are doing something like this in `setup.cfg`:

```
  tensorflow-macos==2.7.0;platform_system=="Darwin" and platform_machine=="arm64"
   # In other places
   tensorflow==2.7.0;platform_system!="Darwin" or platform_machine!="arm64"
```

It seems that tensorflow needs a specific package for M1-based macbooks.
Just for you to know for future updates.